docs/samples/deployments/index.md

```diff
@@ -4,3 +4,4 @@
 - [Kubernetes](k8s/index.md)
 - [Spark/Azure Databricks](spark/index.md)
 - [Azure Data Factory](data-factory/index.md)
+- [Streamlit (simple demo app)](streamlit/index.md)
```

Comment:
sure its a "deployment" and not "python" ?

Reply:
Not sure, it's neither here or there... WDYT?

Reply:
Moved to python